package.json

```diff
@@ -31,6 +31,11 @@
     "configuration": {
       "title": ":emojisense:",
       "properties": {
+        "emojisense.enableEmojisInAllLanguages": {
+          "type": "boolean",
+          "description": "Lets you easily enable emoji decorators in ALL languages",
+          "default": true
```

Comment:
Default this to false instead. Emoji suggestions will get in the way in many languages

Reply:
Should I also default the `enableEmojisInAllLanguages` to `false` in `LanguageConfig` & `Configuration`? 

https://github.com/mattbierner/vscode-emojisense/pull/14/files/9253933e2409d2dd1214dfdb2928e806fafd41a2#diff-8b01d5598b81332eb62385d26f3ead22R8

Reply:
Yes, when you install the extension the default should be `false`. `enableEmojisInAllLanguages` should be an opt-in setting.

On further thought, a cleaner way to express this may be to support using `*` with the `emojisense.languages` setting. That way you can enable or disable completions and decorators separately and we don't need another setting:

```json
"emojisense.languages": {
    "*": true
}
```

or

```json
    "plaintext": {
        "markupCompletionsEnabled": false,
        "emojiDecoratorsEnabled": true
    }
}
```

Reply:
I definitely agree - I've thought about having the star thingie `*` too, but am kinda new to extension development and it didn't seem to work with either of my approaches. I'm currently busy and won't be able to take care of this, so anyone's free to fork my fork and go from there.